docker-compose-separate-containers.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  nginx:
+    image: nginx
+    container_name: nginx
+    ports:
+      - "80:80"
+    volumes:
+      - /tmp/nginx:/etc/nginx/conf.d
```

Comment:
There must be a volume because dockergen mounts it with `--volumes-from nginx` below. It does not have to be mounted on /tmp but that is how the README documents doing it so I did it that way here as well.